internal/schema/elements.go

```diff
@@ -62,3 +62,101 @@ func GetElementType(e specschema.ElementType) string {
 
 	return ""
 }
+
+// GetElementValueType generates the strings for use within templates for specifying the value types
+// to use with collection (i.e., list, map and set) element types.
+func GetElementValueType(e specschema.ElementType) string {
+	switch {
+	case e.Bool != nil:
+		if e.Bool.CustomType != nil {
+			return e.Bool.CustomType.ValueType
+		}
+		return "types.Bool"
+	case e.Float64 != nil:
+		if e.Float64.CustomType != nil {
+			return e.Float64.CustomType.ValueType
+		}
+		return "types.Float64"
+	case e.Int64 != nil:
+		if e.Int64.CustomType != nil {
+			return e.Int64.CustomType.ValueType
+		}
+		return "types.Int64"
+	case e.List != nil:
+		if e.List.CustomType != nil {
+			return e.List.CustomType.ValueType
+		}
+		return "types.ListType"
```

Comment:
Was this meant to be: `types.List` ?

Same for the others below:
- `types.Map`
- `types.Object`
- `types.Set`

Reply:
Thank you! You're quite right. Have fixed.